app/classifier/tasks/survey/choice.jsx

```diff
@@ -73,6 +72,28 @@ class Choice extends React.Component {
     this.props.onConfirm(this.props.choiceID, this.state.answers);
   }
 
+  handleRadioKeyDown(e) {
+    switch (e.which) {
+      case BACKSPACE:
+      case SPACE:
+        if (e.target.checked) {
+          e.preventDefault();
```

Comment:
This is going to prevent the default action for checkboxes as well as radio buttons, so they can't be toggled off with space.

Reply:
yep, adding ` && (e.target.type === 'radio')` to the conditional appears to fix it